test/oas3/execute/build-request.js

```diff
@@ -0,0 +1,82 @@
+// https://github.com/swagger-api/swagger-js/issues/1116
+import { buildRequest } from '../../../src/execute';
+
+describe('test accept header for oas 3', () => {
```

Comment:
```suggestion
describe('buildRequest - OAS 3.0.x', () => {
```

Reply:
Making it consistent with https://github.com/swagger-api/swagger-js/blob/master/test/swagger2/execute/build-request.js#L3